spec/game_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'spec_helper'
+
+describe Game do
+  context 'Testing Games' do
+    before(:each) do
+      @game = Game.new('2011-06-12', 'Y', '2014-05-12')
+    end
+
+    it 'Should be an instance of the Game class' do
+      expect(@game).to be_instance_of Game
+    end
+
+    it 'Should return true as the game is a multiplayer' do
+      expect(@game.multiplayer).to eq 'Y'
+    end
+
+    it 'Should add one game to the collection' do
+      expect(@game.last_played_at).to eq '2014-05-12'
```

Comment:
In line `18` you're checking if the `@game.last_played_at` property is equal to '2014-05-12`, but the message doesn't seem to be quite clear in line `17`, don't you think it'll be better to explain the expectation which is about to happen in line `18`.